packages/apollo/src/components/OrdererDetails/ChannelParticipationDetails.js

```diff
@@ -106,6 +198,355 @@ class ChannelParticipationDetails extends Component {
 		});
 	};
 
+	getChannelDetails = cb => {
```

Comment:
- i don't want this code in two places, refactor this code so that `ChannelDetails.js` and `ChannelParticipationDetails.js` call 1 function that does the work in this block